src/constants/dispenser.ts

```diff
@@ -0,0 +1,6 @@
+const DISPENSER_STATE = {
+  EMPTY: 'empty',
+  MAKING: 'making',
+};
```

Comment:
```suggestion
const DISPENSER_STATE = {
  EMPTY: 'empty',
  MAKING: 'making',
} as const; 
```

요고 하리 아래에는 적용했는데 여기는 안한거보면,  깜빡한 것 같은데! 아니면, 이 상수 객체에만 as const 를 안붙인 이유가 따로 있을까요?

Reply:
제가 어디에 했나 생각이 안 나서 보고 왔습니다 🤣 theme 정의할 때 썼었네용,,
이 부분은 제가 특별한 기준 없이 어디에는 쓰고 어디에는 안 쓰고 했던 것 같아요!
알려주셔서 감사합니다 :) 앞으로 상수 정의할 때는 붙이는 방향으로 기준을 세워볼게용 😆 [commit](https://github.com/woowacourse-study/2022-looking-for-the-sound-of-an-object/pull/18/commits/8790e28003247879107d1269c0acfe26480ef0ea)